distro/run/qa/pom.xml

```diff
@@ -44,7 +44,7 @@
       <dependency>
         <groupId>org.seleniumhq.selenium</groupId>
         <artifactId>selenium-java</artifactId>
-        <version>${version.selenium}</version>
+        <version>4.10.0</version>
```

Comment:
🔧 Can we create a follow-up ticket to streamline and consolidate all selenium dependencies to the same version? So can we bump the selenium version only once in the parent pom?
Cf. https://github.com/search?q=repo%3Acamunda%2Fcamunda-bpm-platform%20version.selenium&type=code

Reply:
Sure, i've created [this](https://github.com/camunda/camunda-bpm-platform/issues/4020) ticket to follow-up on this point.